conclave-common/src/main/kotlin/com/r3/conclave/common/internal/PluginUtils.kt

```diff
@@ -11,5 +11,6 @@ object PluginUtils {
     const val GRAMINE_SGX_TOKEN = "java.token"
     /** This contains the enclave's [SgxEnclaveCss], also known as the `SIGSTRUCT`. */
     const val GRAMINE_SIGSTRUCT = "java.sig"
+    const val GRAMINE_SECCOMP = "gramine-seccomp.json"
```

Comment:
This file will need to be passed in the bundle, for the same reasons of the integration tests @shamsasari you fixed.

Reply:
See comment below, but I don't seewhy we need to copy this file from the enclave plugin to the host. If it's needed by both the plugin and host, then put it in `conclave-common`.

Reply:
Yes, technically this is needed only by the host, so can embed this there.

Reply:
Moved to the host side